src/client/js/Panels/MetaEditor/MetaEditorControl.js

```diff
@@ -1040,6 +1049,29 @@ define(['js/logger',
     /*  END OF --- DISPLAY META CONTAINMENT RELATIONS AS A CONNECTION FROM PARENT TO OBJECT       */
     /**********************************************************************************************/
 
+    MetaEditorControl.prototype._processNodeMixins = function (gmeID) {
```

Comment:
Some logging statements would be nice. If nothing could go wrong use at least one `logger.debug`.